types/etherport-client/index.d.ts

```diff
@@ -0,0 +1,21 @@
+/// <reference types="node"/>
+
+import type {EventEmitter} from "events"
+
+export interface EtherPortClientConstructorArgs {
+    host: string;
+    port: number;
+    reconnectTimeout?: number;
+}
+
+// tslint:disable-next-line:no-unnecessary-class
+export class EtherPortClient extends EventEmitter {
+    path: string;
+    name: string;
+    host: string;
+    port: number;
+
+    constructor(options: EtherPortClientConstructorArgs);
+}
+
+export function chainSerialPorts(clientPort: any, serverPort: any): void
```

Comment:
The Serial stream that `chainSerialPorts` expects [extends `Duplex`](https://github.com/serialport/node-serialport/blob/7d059bd4cac81830f7ba797ed01ccc1b30d389e3/packages/stream/lib/index.ts#L70), so something like this should work (after importing `Duplex` from `node:stream`)

Something like this should work:

```suggestion
export function chainSerialPorts(clientPort: EtherPortClient, serverPort: Duplex): void
```

Reply:
@MysteryBlokHed we should be good now